lib/reporters/json.js

```diff
@@ -78,7 +78,16 @@ function JSONReporter(runner, options = {}) {
 
     runner.testResults = obj;
 
-    var json = JSON.stringify(obj, null, 2);
+    var optionsIndentSize;
+    if (options.reporterOption && 'indentSize' in options.reporterOption) {
+      optionsIndentSize = options.reporterOption.indentSize;
+    } else {
+      optionsIndentSize = 2; // Default Indentation size
+    }
+
+    var indentSize = parseInt(optionsIndentSize, 10); // Cast string to int
```

Comment:
[Bug] What if someone passes in `'\t'`? I'd think it'd be nice to allow that. And no real reason not to, right?

```suggestion
```

...which makes me thing: how about just calling this `options.indentation`?

Reply:
I agree. Handling tabs is a good addition